iceoryx_binding_c/source/c_chunk.cpp

```diff
@@ -43,6 +43,31 @@ const void* iox_chunk_header_to_user_header_const(const iox_chunk_header_t* cons
     return reinterpret_cast<const ChunkHeader*>(chunkHeader)->userHeader();
 }
 
+uint32_t iox_chunk_header_user_chunk_size(const iox_chunk_header_t* const chunkHeader)
+{
+    return reinterpret_cast<const ChunkHeader*>(chunkHeader)->chunkSize();
```

Comment:
Although it is not done in the other functions, could you please add an `iox::cxx::Expects(chunkHeader != nullptr);`

For the existing code this is tracked with issue #1106